homeassistant/components/braviatv/config_flow.py

```diff
@@ -133,16 +172,44 @@ async def async_step_authorize(
         assert self.client
 
         try:
-            await self.client.pair(self.client_id, self.nickname)
+            await self.client.pair(client_id, nickname)
         except BraviaTVError:
             return self.async_abort(reason="no_ip_control")
 
         return self.async_show_form(
-            step_id="authorize",
+            step_id="pin",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(CONF_PIN, default=""): str,
```

Comment:
Why do we need an empty string as default?

Reply:
True, is not needed 👍. Fixed.